src/components/clusterConfiguration/utils.ts

```diff
@@ -64,6 +64,7 @@ export const getInitialValues = (
   ingressVip: cluster.ingressVip || '',
   pullSecret: '',
   sshPublicKey: cluster.sshPublicKey || '',
+  imageSshPublicKey: cluster.imageInfo.sshPublicKey || '',
```

Comment:
I think this should not be set in initialValues as initial values are set only when the form is mounted. Instead we should pass the cluster.imageInfo.sshPublicKey as prop to the `SshClusterKey` component